### ISSUE TITLE
(useVirtualList): scrollbar is always show, even thought mouse is not on the list scrolling

### ISSUE THREAD

**Issue by joyexpr** [vueuse/vueuse#2536]
### Describe the bug

scrollbar is always show, even thought mouse is not on the list scrolling

expectation:  when mouse out of list, the scrollbar will auto hidden

### Reproduction

 just see the demo on page: https://vueuse.org/core/useVirtualList/ 

### System Info

```Shell
System:
    OS: macOS 10.15.7
    CPU: (4) x64 Intel(R) Core(TM) i5-4258U CPU @ 2.40GHz
    Memory: 56.52 MB / 8.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 14.21.1 - ~/.nvm/versions/node/v14.21.1/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v14.21.1/bin/yarn
    npm: 6.14.17 - ~/.nvm/versions/node/v14.21.1/bin/npm
  Browsers:
    Chrome: 108.0.5359.98
    Firefox: 86.0
    Safari: 15.6.1
  npmPackages:
    @vueuse/core: ^9.6.0 => 9.6.0 
    vue: ^3.2.45 => 3.2.45
```

### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vueuse/vueuse/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vueuse.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/vueuse/vueuse/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a VueUse issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to https://github.com/vuejs/core instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vueuse/vueuse/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by wheatjs** [comment]
I don't believe this is how it is supposed to work. The virtual scroller still uses the native scrollbar. On some platforms like MacOS it may automatically hide the scrollbar when not active, but that is not related to the functionality of useVirtualList
